assets/base.css

```diff
@@ -3120,3 +3120,43 @@ details-disclosure > details {
 .rte blockquote > * {
   margin: -0.5rem 0 -0.5rem 0;
 }
+
+/* Image mask global styles */
+
+.shape--mask {
```

Comment:
When I saw `shape--mask` and `shape--arch`, I assumed that these classes would be on the same level. The `--modifier` naming structure we're using here suggests that these are variations of a `shape` element.

Do you think we'll expand these class to eventually match this (expected) BEM convention or should we consider renaming this `shape-arch` or `shape__arch` to better align with our current naming pattern?